pkg/operator/target_config_reconciler.go

```diff
@@ -99,8 +98,14 @@ func (c TargetConfigReconciler) sync() error {
 		return fmt.Errorf("descheduler should have an interval set")
 	}
 
-	if err := validateStrategies(descheduler.Spec.Strategies); err != nil {
-		return err
+	// TODO(@damemi): Remove this check and related functions when Strategies is removed
+	if len(descheduler.Spec.Strategies) > 0 {
```

Comment:
I think the order should be new field first, with fallback to deprecated field, no?

Reply:
Yeah, that's a good point. Will fix in a follow up PR